beacon-chain/blockchain/service.go

```diff
@@ -215,6 +217,7 @@ func (s *Service) Start() {
 	}
 	s.spawnProcessAttestationsRoutine()
 	go s.runLateBlockTasks()
+	go s.updateBlockWithInclusionListRoutine()
```

Comment:
What makes it better to have a routine here instead of updating a payload when a validator has a proposer role? Easier implementation?

Reply:
The routine is a no op unless the validator has a proposer role regardless